Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs

```diff
@@ -92,6 +92,8 @@ public partial class PatientDetailsPage : FluxorComponent
 
         private string? StatusWarning => this.Patient == null ? null : FormattingUtility.FormatPatientStatus(this.Patient.Status);
 
+        private bool ShowStatusWarning => !(this.UserHasRole(Roles.Support) && this.Patient?.Status == PatientStatus.NotUser);
```

Comment:
I think this is more understandable if the brackets are done away with, e.g.

`this.UserHasRole(Roles.Admin) || this.UserHasRole(Roles.Reviewer) || this.Patient?.Status != PatientStatus.NotUser`

It also better handles the case where a user has both the Admin and Support roles.

Reply:
Does that handle when user has AdminUser role and the status is NotUser?

Reply:
Admin and Reviewer role can see the status warning.  Other roles can see it only when the status is something other than "Not User".

Reply:
Changed.